### PR TITLE
Feature/test telegram

### PR DESCRIPTION
**Pull request by Lex-23**

### CODE REVIEW

File: cryptostock/celery_tasks/broker_notification_tasks.py [hunk 1]
```diff
@@ -15,8 +16,10 @@ def notification_success_offer(offer_id):
         f"bought {offer.count} {offer.deal.asset.name} in {offer.timestamp}."
     )
     recipient = [str(offer.deal.broker.owner.email)]
-
-    print(f"{subject}\n{message}\n{recipient}")
+    notification(
+        chat_id=offer.broker.telegram_chat_id,
+        text=f"{subject}\n{message}\nbuyer email: {recipient}",
+    )
```
Comment (vadim-denisov) [inline cryptostock/celery_tasks/broker_notification_tasks.py:22]: I think signature for `notify` should be producer independent. 
```suggestion
    notification(
        broker=offer.broker,
        text=f"{subject}\n{message}\nbuyer email: {recipient}",
    )
```

File: cryptostock/utils/telegram_bot.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import logging
```
[the remark this answers is not in the document]
Comment (vadim-denisov) [reply cryptostock/utils/telegram_bot.py:1]: But in this case we cannot use `notification`.

File: cryptostock/celery_tasks/broker_notification_tasks.py (hunk 1 above)
Comment (vadim-denisov) [reply cryptostock/celery_tasks/broker_notification_tasks.py:22]: Also notification is async method.

Comment (Lex-23) [reply cryptostock/celery_tasks/broker_notification_tasks.py:22]: but for this function I need exactly id

**Pull request closed without merging**